### DISCUSSION TITLE
Chat on Matrix in #rnote:matrix.org

### DISCUSSION THREAD

**Discussion by flxzt** [flxzt/rnote#110] [Announcements]
Join us in the [#rnote:matrix.org](https://matrix.to/#/#rnote:matrix.org) chat room!
You can ask questions as a user, discuss the development or chat about the state of the world. :)

**Comment by niklasw** [comment]
Hi, I just ran rnote from a flatpak. Extraordinary convenient and kinda modern tool, in my view. Thanks!

So, I'd like to do this: Take a screenshot, putting a png in the clipboard and automatically launch rnote with the screenshot as a background/imported image. This would become a nice tool for "Drawing on screen" during e.g. online-meetings.

But, I cannot find any command line arguments to rnote that'd allow this. Are there any?
